modules/rules_multitool/metadata.json

```diff
@@ -1,17 +1,12 @@
 {
     "homepage": "https://github.com/theoremlp/rules_multitool",
-    "maintainers": [
-        {
-            "email": "123787712+mark-thm@users.noreply.github.com",
-            "github": "mark-thm",
-            "name": "Mark Elliot"
-        }
-    ],
+    "maintainers": [],
```

Comment:
not sure why this got wiped out by the app, there's a [fixedReleaser](https://github.com/bazel-contrib/publish-to-bcr/blob/main/templates/README.md#optional-configyml) block in https://github.com/theoremlp/rules_multitool/blob/main/.bcr/config.yml

Reply:
ah nvm I see